docs/quickstart.md

```diff
@@ -19,14 +19,21 @@ Some of the roles used in the playbook use root privileges on the managed node,
 you will need to provide the password for the managed node user.
 
 ```
-ansible-playbook playbooks/example-use/playbook.yml -u <managed_node_username> --ask-become-pass
+ansible-playbook playbooks/example-use/playbook.yml -u <managed_node_username> --ask-become-pass -i <pulp-server-name>,
```

Comment:
We should probably update these 2 placeholder strings to be consistent. managed-node == pulp-server.

They exist in other places though.

Reply:
You need to update this too:
https://pulpproject.org/ansible-installer/#using-the-example-playbook

It's under:
https://github.com/pulp/pulpcore/tree/main/docs

Reply:
For the `pulpproject.org` documentation: https://github.com/pulp/pulpproject.org/pull/432